pkg/resources/reconciler.go

```diff
@@ -14,7 +14,6 @@ import (
 	oauthv1 "github.com/openshift/api/oauth/v1"
 	operatorsv1alpha1 "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
 
-	corev1 "k8s.io/api/core/v1"
 	v1 "k8s.io/api/core/v1"
```

Comment:
Was `corev1` not the better name here as there are other v1's such as `oauthv1` & `metav1` with in the same file.